src/Workspaces/Remote/Core/Services/RemoteWorkspace.cs

```diff
@@ -57,7 +57,6 @@ public Solution AddSolution(SolutionInfo solutionInfo)
             lock (_gate)
             {
                 this.OnSolutionAdded(solutionInfo);
-                this.UpdateReferencesAfterAdd();
```

Comment:
Isn't the answer for this question "write a test to cover this scenario?" :smile: